portfolio/src/main/webapp/index.html

```diff
@@ -7,13 +7,98 @@
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
+      <!-- <img src="sunset.webp" alt="sunset Picture" -->
+      <!-- <p style="background-image: url('sunset.webp');"> -->
+    <div id="content" ">
+      <div id="whiteCenter">
```

Comment:
[3.1.9](https://google.github.io/styleguide/htmlcssguide.html#id_Attributes): Where id attributes are strictly required, always include a hyphen in the value

```suggestion
      <div id="white-center">
```

Here and elsewhere.